pkg/controller/cassandra/cassandra_controller.go

```diff
@@ -167,7 +167,7 @@ type ReconcileCassandra struct {
 }
 
 var cassandraInit2CommandTemplate = template.Must(template.New("").Parse(
-	"keytool -keystore /etc/keystore/server-truststore.jks -keypass {{ .KeystorePassword }} -storepass {{ .TruststorePassword }} -list -alias CARoot -noprompt;" +
+	"echo \"$POD_IP $(cat /etc/hostname)\" >> /etc/hosts && keytool -keystore /etc/keystore/server-truststore.jks -keypass {{ .KeystorePassword }} -storepass {{ .TruststorePassword }} -list -alias CARoot -noprompt;" +
```

Comment:
What was the hostname in this case?

It looks like a hack for me to fix some environment/server configuration issues. I don't like that.